docs/src/modules/utils/helpers.ts

```diff
@@ -165,7 +165,6 @@ export function getDependencies(
     '@material-ui/icons': getMuiPackageVersion('icons', muiCommitRef),
     '@material-ui/lab': getMuiPackageVersion('lab', muiCommitRef),
     '@material-ui/styled-engine': getMuiPackageVersion('styled-engine', muiCommitRef),
-    '@material-ui/styled-engine-sc': getMuiPackageVersion('styled-engine-sc', muiCommitRef),
```

Comment:
How would I use `@material-ui/styled-engine-sc` from a PR in a codesandbox now?

Reply:
My assumption is that all the demos should use `@material-ui/styled-engine` over `@material-ui/styled-engine-sc`, so this case shouldn't happen. Why this assumption? Because `@material-ui/styled-engine`  is meant to be the canonical import.